NEWS.md

```diff
@@ -1,8 +1,34 @@
 # scales (development version)
 
+## Breaking changes
```

Comment:
Can you please leave these headings off? I add them just before release. Otherwise, looks good!

Reply:
Ah, sorry for that! I wasn't sure how else to flag them as a breaking change, but it sounds like you'll do this later. Thanks!